capture/genBitmaps.js

```diff
@@ -158,7 +158,7 @@ function processScenario (casper, scenario, scenarioOrVariantLabel, scenarioLabe
         scenario.removeSelectors.forEach(function (o, i, a) {
           casper.evaluate(function (o) {
             Array.prototype.forEach.call(document.querySelectorAll(o), function (s, j) {
-              s.style.display = 'none';
+                s.remove();
```

Comment:
https://dom.spec.whatwg.org/#dom-childnode-remove - i tested this with Backstop v2. it seems to be working